lib/Service/SampleConversationsService.php

```diff
@@ -0,0 +1,123 @@
+use OCP\IConfig;
+use OCP\IL10N;
+use OCP\IURLGenerator;
+use OCP\IUser;
+use OCP\IUserManager;
+use OCP\PreConditionNotMetException;
+use OCP\Security\ISecureRandom;
+
+class SampleConversationsService {
+	public function __construct(
+		protected IConfig $config,
+		protected IUserManager $userManager,
+		protected Manager $manager,
+		protected ChatManager $chatManager,
+		protected RoomService $roomService,
+		protected AvatarService $avatarService,
+		protected ParticipantService $participantService,
+		protected ISecureRandom $secureRandom,
+		protected IRootFolder $rootFolder,
+		protected IURLGenerator $url,
+		protected ITimeFactory $timeFactory,
+		protected IL10N $l,
+	) {
+	}
+
+	public function initialCreateSamples(string $userId): void {
+		$created = $this->config->getUserValue($userId, 'spreed', 'samples_created');
+		if ($created !== '') {
+			return;
+		}
+
+		$this->config->setUserValue($userId, 'spreed', 'samples_created', $this->timeFactory->now()->format(\DateTime::ATOM));
+
+		$user = $this->userManager->get($userId);
+		if (!$user instanceof IUser) {
+			throw new \InvalidArgumentException('User not found');
+		}
+
+		$this->defaultSampleConversation($user);
+	}
+
+	protected function defaultSampleConversation(IUser $user): void {
+		$userFolder = $this->rootFolder->getUserFolder($user->getUID());
+		$room = $this->roomService->createConversation(
+			Room::TYPE_GROUP,
+			$this->l->t('Let´s get started!'),
+			$user,
+			'sample',
+			$user->getUID()
+		);
+
+		$this->avatarService->setAvatarFromEmoji($room, '💡', '0082c9');
+
+		$messages = [
+			$this->l->t('## ⚙️ Manage conversation settings
+
+In the conversation menu, you can access various settings to manage your conversations, such as:
+* Edit conversation info
+* Manage notifications
+* Apply various moderation rules
+* Configure access and security
+* Enable bots
+* and more!'),
+			$this->l->t('## 🎨 Format texts to create rich messages
+
+In Nextcloud Talk, you can use markdown commands to format your messages. For example, apply **bold** or *italic* formatting, or `highlight texts as code`. You can even create tables and add headings to your text.
+
+Need to fix a typo or change formatting? Edit your message by clicking "Edit message" in the message menu.'),
+			$this->l->t('## 🔗 Add attachments and links
+
+Attach any files from your Nextcloud Hub via "+" button. Link files and items from different apps in your messages. Some apps support interactive widgets, for example the Text app.')
+			. "\n\n". '{FILE:Readme.md}',
+			'{REPLY}' . $this->l->t('You can reply to messages, forward them to other chats and people, or copy message content.'),
+			$this->l->t('## 💡 Do more with Smart Picker
+
+You can configure Smart Picker apps to attach various content to your messages like GIFs, map locations or items from other core Nextcloud apps, or use various AI features.
+
+To get started, access Smart Picker via attachment menu or simply type "/".'),
+		];
```

Comment:
- Link to the user docs somewhere? https://docs.nextcloud.com/server/latest/user_manual/en/talk/index.html
- Reactions?

Reply:
I had skipped the reaction message, added it now.

> Link to the user docs somewhere? https://docs.nextcloud.com/server/latest/user_manual/en/talk/index.html

cc @jancborchardt what do you think?

Reply:
Clarified in chat and added to description for now